server/mediapublic/tests/gabbits/people.yaml

```diff
@@ -0,0 +1,47 @@
+defaults:
+    ssl: False
+    request_headers:
+      # JSON ALL DAY
+      content-type: application/json
+
+tests:
+  - name: Get Peoples
+    url: /people
+    response_json_paths:
+      $.people: []
+
+  - name: Create a person incorrectly
+    status: 400
+    method: POST
+    url: /people
+    data:
+      dummy: bad data
+
+  - name: Create a person correctly
+    status: 201
+    method: POST
+    url: /people
+    data:
+      first: 'Joe'
+      last: 'Smith'
+      address_0: '47 Something St.'
+      address_1: 'Apt. 9'
+      city: 'Rochester'
+      state: 'NY'
+      zipcode: '14623'
+      phone: '585-123-4567'
+      fax: '585-098-7654'
+      primary_website: 'http://www.joesmithissupercool.io/'
+      secondary_website: ' '
+      twitter: '@joesmithandstuff'
+      facebook: ' '
```

Comment:
Why does this have to be sent at all? We shouldn't require the keys for these social accounts to be included on create.